kernel/genarch/src/drivers/pl011/pl011.c

```diff
@@ -59,9 +59,13 @@ static void pl011_uart_putwchar(outdev_t *dev, wchar_t ch)
 {
 	pl011_uart_t *uart = dev->data;
 
-	if (!ascii_check(ch)) {
+	/* If the userspace owns the console, do not output anything. */
+	if (uart->parea.mapped && !console_override)
+		return;
+
+	if (!ascii_check(ch))
```

Comment:
I quite dislike mismatched branch bracing.
We have quite a few of those in other places, so obviously that's just me nitpicking,
but I wonder if this style is something we generally consider ok, or if it would make sense
to curb it using ccheck?

Reply:
This is not completely clear to me. What is meant by "mismatched branch bracing" here and how should be this snippet ideally formatted?

Reply:
What I mean is when one branch of the `if` is braced, but the other branch isn't.
For example
```c
if (condition) {
    //... code
} else
   one_statement();
```
or
```c
if (condition)
    one_statement();
else {
    //... code
}
```

In my opinion, either both branches should be braced (even in the single line case) or neither.
But that may be just me.

Reply:
I see.